exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -272,17 +272,19 @@
       }
        <#if model.shouldShowAdminInfo() || !model.isAuthEnabled()>
           function shutdown(button) {
-              var requestPath = "/gracefulShutdown";
-              var url = getRequestUrl(requestPath);
-              var result = $.ajax({
-                    type: 'POST',
-                    url: url,
-                    contentType : 'text/plain',
-                    complete: function(data) {
-                        alert(data.responseJSON["response"]);
-                        button.prop('disabled',true).css('opacity',0.5);
-                    }
-              });
+              if (confirm("Click ok to shutdown")) {
```

Comment:
Message should be more of like` "Are you sure you want to shutdown Drillbit running on + location.host + node ?"`